### ISSUE TITLE
CatalogueCompiler invalidates whole cache

### ISSUE THREAD

**Issue by hubipe** [Kdyby/Translation#136]
CatalogueCompiler has method invalidateCache, but when called, it destroys complete cache, not just it's own namespace. This behaviour is unexpected. CatalogueComplier should invalidate just it's own namespace.

**Comment by fprochazka** [comment]
@hubipe you're right. This will be changed in next major, when I drop the unnecesary custom implementations.
